extensions/aimlab/src/views/profile.tsx

```diff
@@ -0,0 +1,54 @@
+import { Preferences } from "../types/preferences.types";
+import { List, getPreferenceValues } from "@raycast/api";
+import { useEffect, useState } from "react";
+import PlayerComponent from "../components/Player";
+import useProfile from "../hooks/useProfile";
+
+type PropTypes = {
+  username: string;
+};
+
+const UserProfile = ({ username }: PropTypes) => {
+  const preferences = getPreferenceValues<Preferences>();
```

Comment:
Note that you can use the types from `raycast-env.d.ts` to automatically get preferences types: https://developers.raycast.com/changelog#new-3.